.rubocop.yml

```diff
@@ -44,6 +44,10 @@ Style/FrozenStringLiteralComment:
 Style/AsciiComments:
   Enabled: false
 
+# We are not strict to the point that we want to enforce super-short `it '…' do … end` blocks in our specs.
+RSpec/ExampleLength:
+  Enabled: false
```

Comment:
At the same time, it's useful to have some kind of warning when you end up with an example that is _too_ long.

Now, how long is "_too_ long"? Well... that's a good question 😅

Maybe it's fair to say that we have bigger fishes to fry right now and that if we'll get to the point where "_too_ long" specs really pop out as an issue, then it means we'll have achieve a lot of our codebase code quality improvement goals 😄 👍

Reply:
See also https://github.com/wordpress-mobile/release-toolkit/issues/248.

As per my previous comment, I'm happy to got with this change and close the issue for the time being.

If you're reading this and that sounds good to you, feel free to close the issue.

Reply:
Yeah I almost just raised the max value instead of entirely disabling it.

I could also have set this rule in `.rubocop_todo.yml` instead of `.rubocop.yml` to signal the real intent, which really was "yeah that's a good rule, but we have other fishes to fry first so we don't want to burn too much time on this one in priority while there are other more important violations to fix first" 😄 

My plan is to fix all the violations incrementally, starting with what we deem the most "important" rules/conventions to respect first (the ones that really make the code consistent, avoid security issues, and make the code easier to navigate by having e.g. file names matching the name of the class they implement, etc), and slowly and incrementally go through the exceptions in `.rubocop_todo.yml` one by one until the TODO file is empty and we got rid of all those "we'll deal with that later" exceptions. Once we get to that point, and have a real clean codebase, then it can be time to revisit the rules we decided to enable/disable back then and get the finishing touches to iron things out 😉